---
float32 actual_current
